public/locales/en/translation.json

```
@@ -98,7 +98,20 @@
   "Schedule Export": "Schedule Export",
   "Generate a high-fidelity cryptographic ledger of your current courses.": "Generate a high-fidelity cryptographic ledger of your current courses.",
   "SYCHRONIZING...": "SYCHRONIZING...",
-  "Download PDF Manifest": "Download PDF Manifest",
+  "General": "General",
```

Biome (2.4.13)

[error] The key General was already declared.

(lint/suspicious/noDuplicateObjectKeys)